lib/CXGN/Analysis/AnalysisCreate.pm

```diff
@@ -365,6 +365,10 @@ sub store {
     print STDERR Dumper $analysis_model_type;
     my $model_type_cvterm_id = SGN::Model::Cvterm->get_cvterm_row($bcs_schema, $analysis_model_type, 'protocol_type')->cvterm_id();
 
+    if ($analysis_to_save_boolean eq 'yes' && !$analysis_name) {
```

Comment:
shouldn't this be 
if ( ($analysis_to_save_boolean eq 'yes') && !$analysis_name) { 
(the first comparison needs parentheses)

Reply:
Verified the precedence and this should be ok :-)